Homework_9_Incapsulation_Inheritance_Polymorphism/My_Company/src/com/example/employees/Employee.java

```diff
@@ -0,0 +1,78 @@
+package com.example.employees;
+
+public class Employee {
+
+    private String name;
+    private int age;
+    private int salary;
+    private static int minSalary = 2000;
```

Comment:
despite it's a private constant -> it's still a static constant 
Convention says `MIN_SALARY`

And can be public then, why not, maybe someone would like to take it as a reference

Reply:
Ooops sorry I thought it was final

Anyway looks like a constant according to its semantics

`public static final int MIN_SALARY = 2000;`